site/templates/collection/continue_watching.jet

```diff
@@ -0,0 +1,33 @@
+{{import "./page_collection_item.jet"}}
+{{import "../common/slider.jet"}}
+{{import "./slider-buttons.jet"}}
+
+{{block continueWatchingCollection()}}
+  {{layout := config("default_image_type")}}
+  {{itemsPerRow := layout == "portrait" ? 6 : 4}}
+  <s72-continue-watching
+      hide-if-empty="true"
+      partials-base-path="/partials"
+      partials-classes="swiper-slide"
+      partials-insert-point-selector=".swiper-wrapper"
+      force-login="false"
+      data-item-layout="{{layout}}"
+      data-items-per-page="{{itemsPerRow}}"
+      class="user-continue-watching s72-slider s72-hide">
+    <section class="page-collection" aria-label="{{i18n("wcag_aria_label_continue_watching")}}">
+      <div class="collection-wrapper-container swiper-wrapper-container">
+        <div class="collection-container swiper-container" data-items-per-row="{{itemsPerRow}}" data-layout="{{layout}}">
+          {{if i18n("continue_watching_slider_header") != ""}}
+          <div class="swiper-title">
```

Comment:
This div block should be indented inside the if.

Reply:
Yeah I think these issues were inherited from the copy-pasta'd wishlist.jet. I'll tidy them up.

I wasn't sure with the indentation whether this was being done deliberately to preserve the indentation of the resulting markup. It's inconsistent throughout core template...

Reply:
It's not supposed to be inconsistent though....

Reply:
This was the before https://github.com/shift72/core-template/blob/develop/site/templates/collection/wishlist.jet
😆